e2e/specs/planner.e2e-spec.ts

```diff
@@ -0,0 +1,194 @@
+    const idDialogDisplayed = await sessionDetailsDialog.isDialogWithTextsDisplayed(...valuesToCheck);
+    expect(idDialogDisplayed).toBeTruthy();
+    await sessionDetailsDialog.close();
+}
+
+describe('Planner, check newly created sessions existence', () => {
+
+    beforeAll(async () => {
+        await loginFlow.relogin();
+        await navigationFlow.gotoPlannerPage();
+        await plannerPage.clickJudgeViewButton();
+        await plannerPage.openDayView();
+        await plannerPage.clickTodayButton();
+
+        today = DateTimeHelper.generateRandomTimeDuringWorkHours(moment());
+        sessionsToCreate = [{
+            id: uuid(),
+            userTransactionId: uuid(),
+            personId: null,
+            roomId: null,
+            duration: 3600,
+            start: today,
+            sessionTypeCode: SessionTypesCodes[SessionTypes.FTRACK_TRIAL_ONLY]
+        },
+            {
+                id: uuid(),
+                userTransactionId: uuid(),
+                personId: JudgesCodes[Judges.AMY_WESSOME],
+                roomId: null,
+                duration: 3600,
+                start: today,
+                sessionTypeCode: SessionTypesCodes[SessionTypes.FTRACK_TRIAL_ONLY]
+            },
+            {
+                id: uuid(),
+                userTransactionId: uuid(),
+                personId: JudgesCodes[Judges.AMY_WESSOME],
+                roomId: RoomCodes[Rooms.COURT_4],
+                duration: 3600,
+                start: today.clone().minute(15),
+                sessionTypeCode: SessionTypesCodes[SessionTypes.FTRACK_TRIAL_ONLY]
+            }, {
+                id: uuid(),
+                userTransactionId: uuid(),
+                personId: JudgesCodes[Judges.JUDGE_LINDA],
+                roomId: null,
+                duration: 3600,
+                start: today.clone().minute(15),
+                sessionTypeCode: SessionTypesCodes[SessionTypes.FTRACK_TRIAL_ONLY]
+            }
+        ];
+    });
+
+    describe('Check if newly created sessions are on planner ', () => {
+        it('Create sessions ', async () => {
+            numberOfVisibleEvents = await plannerPage.getNumberOfVisibleEvents();
+            expect(numberOfVisibleEvents).not.toEqual(0);
+            for (let session of sessionsToCreate) {
+                expect(await API.createSession(session)).toEqual(200);
+                Logger.log('Created session: id: ' + session.id + ' start: ' + session.start.toISOString());
+            }
+        });
+
+        it('Refresh the view ', async () => {
+            await plannerPage.clickNextButton();
+            await plannerPage.clickPrevButton();
+        });
+
+        it('Refreshed view should show new sessions ', async () => {
+            let newNumberOfVisibleEvents = await plannerPage.getNumberOfVisibleEvents();
+            visibleElementsDiff = Math.abs(newNumberOfVisibleEvents - numberOfVisibleEvents);
+            Logger.log('Sessions number, before call: ' +
```

Comment:
Could we put that log statement into 'getNumberOfVisibleEvents' method? No pollution of testcase with log statements and better reusability

Reply:
not this one, as this is supposed to show the diff and otherwise the whole test/check would have to go out of here